aip/general/0127.md

```diff
@@ -73,18 +73,21 @@ message CreateBookRequest {
     should be populated in the request proto. When extracting a [resource
     name][aip-122], the variable **must** include the entire resource name, not
     just the ID component.
+  - URIs **may** use nested fields for their variable names. (Additionally,
+    AIP-134 mandates this for `Update` requests.)
   - URIs **must** use the `*` character to represent ID components, which
     matches all URI-safe characters except for `/`. URIs **may** use `**` as
     the final segment of a URI if matching `/` is required.
-- The `body` key defines which field in the request will be sent as the HTTP
-  body. If the body is `*`, then this indicates that the request object itself
-  is the HTTP body. The request body is encoded as JSON as defined by protocol
-  buffers' canonical [JSON encoding][].
+- The `body` key defines which one field in the request will be sent as the
```

Comment:
suggest s/`one field`/`one top-level field` (I know you clarify this below, but maybe we can include that here too)

Reply:
I decided "single" was better than "one". Added "top-level".